docs/export-format.md

```diff
@@ -25,12 +25,31 @@ The main output file has values from any non-repeating group field found at the
 
 ## Repeat output file
 
-Each repeat group will produce an output CSV file.
+Each repeat group will produce an output CSV file, and the filename will be the composition of the main csv filename and the repeat group's name.
 
-The file names of repeat groups depend on their position inside the model tree of the form. Some examples:
+- If form X has a repeat group Y, then the output file for the repeat group will be named `X-Y.csv`
+- Middle non-repeat groups are ignored. If form X has a non-repeat group Y, which has a repeat group Z, then the output file for the repeat group will be named `X-Z.csv`
```

Comment:
Now this should replace "non-repeat groups" with "groups (including repeats)" or something like that, right?

Reply:
Done! Thanks for noticing that mistake! :)